pkg/kratos/handlers.go

```diff
@@ -248,6 +251,69 @@ func (a *API) handleGetLoginFlow(w http.ResponseWriter, r *http.Request) {
 	_ = json.NewEncoder(w).Encode(flow)
 }
 
+func (a *API) handleCreateRegistrationFlow(w http.ResponseWriter, r *http.Request) {
+	returnTo := r.URL.Query().Get("return_to")
+
+	flow, cookies, err := a.service.CreateBrowserRegistrationFlow(r.Context(), returnTo)
+	if err != nil {
+		a.logger.Errorf("Failed to create registration flow: %v", err)
+		http.Error(w, "Failed to create registration flow", http.StatusInternalServerError)
+		return
+	}
+
+	setCookies(w, cookies)
+	w.WriteHeader(http.StatusOK)
+
+	toMap, _ := flow.ToMap()
+	_ = json.NewEncoder(w).Encode(toMap)
```

Comment:
question: is it ok to ignore the errors? Are we sure that an error can never happen? Perhaps it would be good to handle potential errors just to be proactive for breaking changes.